### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by amirnathani9**

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0.1rem;
+}
+.badge-container{
+    position: relative;
+}
 /* badge width height */
 .badge-wh-1{
```
Comment (xii-vi) [inline components/badge/badge.css:12]: where is width??

File: components/badge/badge.html [hunk 2]
```diff
@@ -50,56 +50,64 @@ <h2 class="text-xl font-bolder">Badge</h2>
...
+            &lt;img src=&quot;../../assests/avatar.jpg&quot; alt=&quot;avatar&quot; class=&quot;avatar-xl image-round&quot;&gt;
+            &lt;span class=&quot;badge badge-wh-3 badge-position-round-tl online-badge-color&quot;&gt;3&lt;/span&gt;
+        &lt;/figure&gt;
+
+        &lt;!-- badge position bottom-right for round icon/avatar --&gt;
+        &lt;figure class=&quot;badge-container&quot;&gt;
+            &lt;img src=&quot;../../assests/avatar.jpg&quot; alt=&quot;avatar&quot; class=&quot;avatar-xxl image-round &quot;&gt;
+            &lt;span class=&quot;badge badge-wh-4 badge-position-round-br primary-badge-color&quot;&gt;5+&lt;/span&gt;
+        &lt;/figure&gt;
+                    </code><button id="button" class="absolute snippet-btn"><i class="fas fa-copy"></i></button></pre>
```
Comment (haarsh24) [inline components/badge/badge.html:114]: Add formatting

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by haarsh24 — APPROVED** [review]
Good to Merge

**Review by zabihshaik — APPROVED** [review]
Looks Good to merge.

File: components/badge/badge.css (hunk 1 above)
Comment (amirnathani9) [reply components/badge/badge.css:12]: this is changed pr that's why width will not be shown

File: components/badge/badge.html (hunk 2 above)
Comment (amirnathani9) [reply components/badge/badge.html:114]: okk

**Pull request merged**